Makefile

```diff
@@ -58,6 +58,11 @@ pkg/rest/zz_docs: $(wildcard pkg/rest/*.go pkg/korrel8r/*.go) $(SWAG)
 lint: $(VERSION_TXT) $(GOLANGCI_LINT) ## Run the linter to find and fix code style problems.
 	$(GOLANGCI_LINT) run --fix
 
+.PHONY: fmt
```

Comment:
go fmt is not necessary. `golangci-lint run --fix` includes the actions of `go fmt` and is faster than running separately.